### PR TITLE
Use utf-8 encoding for readme file in setup.py - closes #147

### LINKED ISSUE #147: Non-ascii characters in README.md can cause install errors
The `ã` character in a contributor's name can cause ascii decoding errors when trying to pip install aloe. The problem is found on the following line of the README.md:
`Copyright (C) <2010-2012> Gabriel Falcão <gabriel@nacaolivre.org>`

The error produced is:
```
     Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-wj0k1i6w/aloe/setup.py", line 19, in <module>
        long_description=readme.read(),
      File "/home/utils/Python-3.6.1/lib/python3.6/encodings/ascii.py", line 26, in decode
        return codecs.ascii_decode(input, self.errors)[0]
    UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 2494: ordinal not in range(128)
```

### PR DESCRIPTION
**Pull request by nikolas**
This should fix the codec error.

I've run into the same problem in CommonMark-py, and solved it in the same way: https://github.com/rtfd/CommonMark-py/blob/master/setup.py#L32

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -2,13 +2,16 @@
...
+from os import path
 from setuptools import setup, find_packages
 
 if __name__ == '__main__':
+    this_directory = path.abspath(path.dirname(__file__))
     with \
             open('requirements.txt') as requirements, \
             open('test_requirements.txt') as test_requirements, \
-            open('README.md') as readme:
+            io.open(path.join(this_directory, 'README.md'), encoding='utf-8') as readme:
```
Comment (koterpillar) [inline setup.py]: Why the use of `this_directory` here but not above?

**Review by koterpillar — COMMENTED** [review]
If this avoids the problem with a particular version of pip, let's put it in. However, can you please add a comment in the code indicating the problematic conditions (e.g. "pip X.Y, python Z.T, fixed in X.W: link to bug)?

File: setup.py (hunk 1 above)
Comment (nikolas) [reply setup.py]: I assumed nothing needs to be changed for the requirements files, but I guess you could add the same changes there as well.

Comment (koterpillar) [reply setup.py]: I mean, why is `this_directory` needed at all?

Comment (nikolas) [reply setup.py]: Maybe it's not - i'm not sure. Feel free to change it.

Comment (merwok) [reply setup.py]: It’s not in the sense that `setup.py` is supposed to always be run from its parent directory.

**Pull request merged**